src/components/10-atoms/textarea/index.react.d.ts

```diff
@@ -17,9 +17,9 @@ export interface AXATextareaProps {
   counterError?: string;
   maxLength?: number;
   className?: string;
-  onFocus?: () => void;
-  onBlur?: () => void;
-  onChange?: () => void;
+  onFocus?: (e?: FocusEvent) => void;
+  onBlur?: (e?: FocusEvent) => void;
+  onChange?: (e: { target: HTMLInputElement }, invalidFormat: Boolean) => void;
```

Comment:
Copy-paste is dangerous... in this case the event target is `HTMLTextAreaElement`, and there is no such thing as the invalidFormat stuff here.

Please adapt.

Reply:
Oh, and can you test this using TypeScript, whether it's ok?

Reply:
I have no clue how to test this inside the patterns library....